apps/tome-api/src/features/card/card.fns.ts

```diff
@@ -1631,4 +1630,513 @@ export const notImplementedCards: DbCard[] = [
+		- Blue: Heal 10 HP. \
+		- Red: Deal Discard another card from the opponent\'s hand. \
+		- Green: Choose a stack and discard the top card of that stack.',
+		effects: {},
+	},
+	{
+		id: '137',
+		name: 'Major Arcana',
+		type: 'field',
+		color: null,
+		description: 'Cards without the word "of" in their names trigger their effects twice.',
+		effects: {},
+	},
+	{
+		id: '138',
+		name: 'Minor Arcana',
+		type: 'field',
+		color: null,
+		description: 'Cards with names that ends with the letter "s" have +5 attack.',
+		effects: {},
+	},
+	// An archetype based on conjuring familiars in the shape of cute animals
+	{
+		id: '139',
+		name: 'Familiar: Cat',
+		type: 'spell',
+		colors: ['blue'],
+		attack: {
+			label: '10',
+			getValue: ({ game, ownerSide }) => {
+				const hasFamiliarInGreen = game.board.players[ownerSide].stacks.green.some(
+					card => card.name.toLowerCase().includes('familiar'),
+				);
+				return hasFamiliarInGreen ? 15 : 10;
+			},
+		},
+		description: 'If you have a "Familiar" card in the GREEN stack, this card has +5 attack.',
+		effects: {},
+	},
+	{
+		id: '140',
+		name: 'Familiar: Owl',
+		type: 'spell',
+		colors: ['green'],
+		attack: 10,
+		description: 'If you have a "Familiar" card in the RED stack, this card has +5 healing. When this card is revealed, if you have a "Familiar" card in the BLUE stack, draw 1 card.',
+		effects: {},
+	},
+	{
+		id: '141',
+		name: 'Familiar: Fox',
+		type: 'spell',
+		colors: ['red'],
+		attack: 10,
+		description: 'When this card is revealed, if you have a "Familiar" card in the BLUE stack, choose a stack and discard the top card of that stack. If you have a "Familiar" card in the BLUE stack, this card has +3 attack.',
+		effects: {},
+	},
+	{
+		id: '142',
+		name: 'Familiar: Rabbit',
+		type: 'spell',
+		colors: ['blue', 'red'],
+		attack: 10,
+		description: 'If you have a "Familiar" card in the GREEN stack this card has +5 healing.',
+		effects: {},
+	},
+	{
+		id: '143',
+		name: 'Familiar: Toad',
+		type: 'spell',
+		colors: ['blue', 'green'],
+		attack: 10,
+		description: 'When this card is revealed, if you have a "Familiar" card in the RED stack, deal 10 damage to the opponent.',
+		effects: {},
+	},
+	{
+		id: '144',
+		name: 'Familiar: Bear',
+		type: 'spell',
+		colors: ['green', 'red'],
+		attack: 10,
+		description: 'If you have a "Familiar" card in the BLUE stack this card has +5 attack.',
+		effects: {},
+	},
+	{
+		id: '145',
+		name: 'Familiar: Wolf',
+		type: 'spell',
+		colors: [],
+		attack: 10,
+		description: 'If you have another "Familiar" card in play, this card has +5 attack.',
+		effects: {},
+	},
+	{
+		id: '146',
+		name: 'Familiar: Bat',
+		type: 'spell',
+		colors: [],
+		attack: 10,
+		description: 'When this card is revealed, if you have another "Familiar" card in play, draw 2 cards and discard 1 card from your hand.',
+		effects: {},
+	},
+	{
+		id: '147',
+		name: 'Familiar Field',
```

Comment:
```suggestion
		name: 'Enchanted forest of familiars',
```
less repetitive title?

Reply:
ideas:
- magical woods of the familiars
- druid’s hut

Reply:
LOVE IT